website/versioned_docs/version-7.0.0/preset-env.md

```diff
@@ -42,9 +42,9 @@ For browser- or Electron-based projects, we recommend using a [`.browserslistrc`
 
 By default `@babel/preset-env` will use [browserslist config sources](https://github.com/ai/browserslist#queries) _unless_ either the [targets](#targets) or [ignoreBrowserslistConfig](#ignorebrowserslistconfig) options are set.
 
-For example, to only include polyfills and code transforms needed for users whose browsers have >0.25% market share (ignoring browsers without security updates like IE 10 and BlackBerry):
+> Please note that if you are relying on `browserslist`'s `defaults` query (either explicitly or by having no browserslist config), please check out the [No targets](#no-targets) section for more info.
```

Comment:
This sentence doesn't sound good to me 🤔 I think that the second "please" disconnects the "Please note that" from what I should note.

Reply:
Yeah... too many edits.

> Please note that if you are relying on `browserslist`'s `defaults` query (either explicitly or by having no browserslist config), you will want to check out the [No targets](#no-targets) section for more info.

?

Reply:
:+1: